apps/omg_watcher/lib/eventer.ex

```diff
@@ -14,7 +14,7 @@
 
 defmodule OMG.Watcher.Eventer do
   @moduledoc """
-  Imperative shell for handling events
+  Imperative shell for handling events. See OMG.API.EventerAPI for the API to the GenServer
```

Comment:
To learn that Eventer is actually only for events emitted to client one needs to read the code.